src/Morphir/Dependency/DAG.elm

```diff
@@ -22,7 +22,7 @@ This level can be used either to derive a topological ordering or to process in
 
 # Querying
 
-@docs incomingEdges, outgoingEdges
+@docs incomingEdges, outgoingEdges, collectReachableNodes
```

Comment:
Minor issue but I think it will break publishing so we should fix it. You didn't update the function name in the docs.